app/code/Magento/InventoryInStorePickupFrontend/view/frontend/web/template/delivery-method-selector.html

```diff
@@ -0,0 +1,12 @@
+<div id="delivery-method-selector" class="step-content">
```

Comment:
Do not use data-bind. https://devdocs.magento.com/guides/v2.3/ui_comp_guide/concepts/magento-bindings.html?itm_source=devdocs&itm_medium=quick_search&itm_campaign=federated_search&itm_term=ko

Reply:
Applied where applicable.